integration/hive/hive-openlineage-hook/integrations/container/cllCaseInsensitivityComplete.json

```diff
@@ -0,0 +1,179 @@
+                  "transformations": [
+                    {
+                      "type": "DIRECT",
+                      "subtype": "TRANSFORMATION",
+                      "description": "",
+                      "masking": false
+                    }
+                  ]
+                }
+              ]
+            },
+            "transactiontype": {
+              "inputFields": [
+                {
+                  "namespace": "file",
+                  "name": "/opt/hive/data/warehouse/test.db/transactions",
+                  "field": "transactiontype",
+                  "transformations": [
+                    {
+                      "type": "DIRECT",
+                      "subtype": "IDENTITY",
+                      "description": "",
+                      "masking": false
+                    }
+                  ]
+                }
+              ]
+            },
+            "totalamount": {
+              "inputFields": [
+                {
+                  "namespace": "file",
+                  "name": "/opt/hive/data/warehouse/test.db/transactions",
+                  "field": "transactionamount",
+                  "transformations": [
+                    {
+                      "type": "DIRECT",
+                      "subtype": "AGGREGATION",
+                      "description": "",
+                      "masking": false
+                    }
+                  ]
+                }
+              ]
+            }
+          },
+          "dataset": [
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/transactions",
+              "field": "submissiondate",
+              "transformations": [
+                {
+                  "type": "INDIRECT",
+                  "subtype": "GROUP_BY",
+                  "description": "",
+                  "masking": false
+                },
+                {
+                  "type": "INDIRECT",
+                  "subtype": "SORT",
+                  "description": "",
+                  "masking": false
+                }
+              ]
+            },
+            {
+              "namespace": "file",
+              "name": "/opt/hive/data/warehouse/test.db/transactions",
+              "field": "transactiontype",
+              "transformations": [
+                {
+                  "type": "INDIRECT",
+                  "subtype": "GROUP_BY",
+                  "description": "",
+                  "masking": false
+                },
+                {
+                  "type": "INDIRECT",
+                  "subtype": "SORT",
+                  "description": "",
+                  "masking": false
+                }
+              ]
+            }
+          ]
+        },
+        "symlinks": {
+          "identifiers": [
+            {
+              "namespace": "hive",
+              "name": "test.monthly_transaction_summary",
+              "type": "TABLE"
```

Comment:
Please define Hive naming in the website: https://openlineage.io/docs/spec/naming/
Also, `hive://test.monthly_transaction_summary` does not seem to uniquely identify the relevant hive instance - can we get something like host:port there?

Reply:
done